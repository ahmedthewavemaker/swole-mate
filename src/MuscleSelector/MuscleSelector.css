.mucscle{
    display: flex;
    
}

a:visited{
    color: blue;
}