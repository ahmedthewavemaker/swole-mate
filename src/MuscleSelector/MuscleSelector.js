import React from 'react';
import {Link} from 'react-router-dom';
import './MuscleSelector.css'


function MuscleSelector() {

    return (
        <div className='muscle'>
            <div><h2><Link to="/exercise/chest">Chest <br />[image]</Link></h2></div>
            <div><h2><Link to="/exercise/back">Back <br />[image]</Link></h2></div>
            <div><h2><Link to="/exercise/shoulders">Shoulders <br />[image]</Link></h2></div>
            <div><h2><Link to="/exercise/arms">Arms <br />[image]</Link></h2></div>
            <div><h2><Link to="/exercise/legs">Legs <br />[image]</Link></h2></div>
            <div><h2><Link to="/exercise/miscellaneous">Miscellaneous <br />[image]</Link></h2></div>

        </div>


    )

}

export default MuscleSelector;