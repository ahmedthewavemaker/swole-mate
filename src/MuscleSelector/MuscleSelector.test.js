import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import MuscleSelector from './MuscleSelector';
import { MemoryRouter } from 'react-router-dom';

describe('MuscleSelector component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<MemoryRouter> <MuscleSelector />
      </MemoryRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<MemoryRouter> <MuscleSelector />
        </MemoryRouter>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });


});
