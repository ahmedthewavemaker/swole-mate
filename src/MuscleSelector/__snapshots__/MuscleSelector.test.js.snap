// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MuscleSelector component renders the UI as expected 1`] = `
Array [
  " ",
  <div>
    <div>
      <h2>
        <a
          href="/exercise/chest"
          onClick={[Function]}
        >
          Chest 
          <br />
          [image]
        </a>
      </h2>
    </div>
    <div>
      <h2>
        <a
          href="/exercise/back"
          onClick={[Function]}
        >
          Back 
          <br />
          [image]
        </a>
      </h2>
    </div>
    <div>
      <h2>
        <a
          href="/exercise/shoulders"
          onClick={[Function]}
        >
          Shoulders 
          <br />
          [image]
        </a>
      </h2>
    </div>
    <div>
      <h2>
        <a
          href="/exercise/arms"
          onClick={[Function]}
        >
          Arms 
          <br />
          [image]
        </a>
      </h2>
    </div>
    <div>
      <h2>
        <a
          href="/exercise/legs"
          onClick={[Function]}
        >
          Legs 
          <br />
          [image]
        </a>
      </h2>
    </div>
    <div>
      <h2>
        <a
          href="/exercise/miscellaneous"
          onClick={[Function]}
        >
          Miscellaneous 
          <br />
          [image]
        </a>
      </h2>
    </div>
  </div>,
]
`;
